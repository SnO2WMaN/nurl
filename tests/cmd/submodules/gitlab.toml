args = [
    "https://gitlab.com/librewolf-community/browser/source",
    "100.0-1",
    "--submodules",
]
