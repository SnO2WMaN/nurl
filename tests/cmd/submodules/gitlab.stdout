fetchFromGitLab {
  group = "librewolf-community";
  owner = "browser";
  repo = "source";
  rev = "100.0-1";
  hash = "sha256-ijqyZAEKyL7V/NFtI3ErHcSnj7VbBFvgCgK9NJren/U=";
  fetchSubmodules = true;
}